test/integration/editor/new.spec.ts

```diff
@@ -51,6 +52,11 @@ test('can create and place new component', async ({ page }) => {
 });
 
 test('can create/delete page', async ({ page, localApp }) => {
+  invariant(
```

Comment:
Maybe we should have a reusable utility for this type of invariant with this type of error message too, since it seems to be pretty common.